modules/income_and_assets/lib/income_and_assets/pdf_fill/forms/va21p0969.rb

```diff
@@ -831,8 +884,42 @@ def expand_trust(trust)
         end
         expanded.merge(overflow)
       end
+
+      ##
+      # Expands and transforms the `unreportedAssets` field in the form data.
+      #
+      # If `unreportedAssets` is present and not empty, sets a flag `unreportedAssets` to 0,
+      # otherwise sets it to 1. Then maps over the assets to apply individual transformations.
+      #
+      def expand_unreported_assets
+        unreported_assets = form_data['unreportedAssets']
+        form_data['unreportedAssets'] = unreported_assets&.length ? 'YES' : 'NO'
```

Comment:
Might be this line - I think it should be `form_data['unreportedAsset'] = unreported_assets&.length ? 'YES' : 'NO'` instead